models/login_source.go

```diff
@@ -140,12 +140,13 @@ func (cfg *OAuth2Config) ToDB() ([]byte, error) {
 
 // LoginSource represents an external way for authorizing users.
 type LoginSource struct {
-	ID            int64 `xorm:"pk autoincr"`
-	Type          LoginType
-	Name          string          `xorm:"UNIQUE"`
-	IsActived     bool            `xorm:"INDEX NOT NULL DEFAULT false"`
-	IsSyncEnabled bool            `xorm:"INDEX NOT NULL DEFAULT false"`
-	Cfg           core.Conversion `xorm:"TEXT"`
+	ID                            int64           `xorm:"pk autoincr"`
+	Type                          LoginType
+	Name                          string          `xorm:"UNIQUE"`
+	IsActived                     bool            `xorm:"INDEX NOT NULL DEFAULT false"`
+	IsSyncEnabled                 bool            `xorm:"INDEX NOT NULL DEFAULT false"`
+	IsLdapPublicSSHKeySyncEnabled bool            `xorm:"INDEX NOT NULL DEFAULT false"`
```

Comment:
Also this option (if really needed) should not be added to Login source as it is common to all Login sources but to LDAP `Source` type